server/webapp/WEB-INF/rails.new/spec/controllers/admin/plugins/plugins_controller_spec.rb

```diff
@@ -30,10 +29,18 @@
       upload_plugin_path.should == "/admin/plugins"
     end
 
+    it "should resolve_the_route_for_get plugin settings" do
+      {:get => "/admin/plugins/settings/plugin.id"}.should route_to(:controller => "admin/plugins/plugins", :action => "edit_settings", :plugin_id => "plugin.id")
+      edit_settings_path(:plugin_id => 'plugin.id').should == "/admin/plugins/settings/plugin.id"
+    end
+
+    it "should resolve_the_route_for_update plugin settings" do
+      {:post => "/admin/plugins/settings/plugin.id"}.should route_to(:controller => "admin/plugins/plugins", :action => "update_settings", :plugin_id => "plugin.id")
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.